common/middleware/getObjectiveData.js

```diff
@@ -0,0 +1,79 @@
+const { logger } = require('../logging/logger')
+const {
+  getSentencePlanObjective,
+  getSentencePlanNeeds,
+  getMotivations,
+  getInterventions,
+} = require('../data/sentencePlanningApi')
+const { getYearMonthFromDate, getStatusText, RESPONSIBLE_LIST } = require('../utils/util')
+
+const catchAndReThrowError = (msg, error, res) => {
```

Comment:
Should we move this to utils (or similar) and reuse the living whatnot out of it?